udata/core/dataset/events.py

```diff
@@ -49,7 +49,8 @@ def publish(url, document, resource_id, action):
         "dataset_id": str(document.id),
         "document": resource,
     }
-    r = requests.post(url, json=payload)
+    headers = {"Authorization": f"Bearer {current_app.config['RESOURCES_ANALYSER_API_KEY']}"}
+    r = requests.post(url, json=payload, headers=headers)
```

Comment:
For my information, this method is used for all actions (creating, updating and deleting) about a resource on hydra, right?

Reply:
yep, it's called by
- `publish_added_resource_message`
- `publish_updated_resource_message`
- `publish_removed_resource_message`
in the same file.